core/state/statedb.go

```diff
@@ -88,14 +89,18 @@ func (self *StateDB) Refund(gas *big.Int) {
 	self.refund.Add(self.refund, gas)
 }
 
-/*
- * GETTERS
- */
-
 func (self *StateDB) HasAccount(addr common.Address) bool {
 	return self.GetStateObject(addr) != nil
 }
 
+func (self *StateDB) Exist(addr common.Address) bool {
+	return self.GetStateObject(addr) != nil
```

Comment:
Also would recommend to ensure ```GetStateObject``` is idempotent - currently it will create the object if it doesn't exist. If this cannot easily be accomplished, ```Exist``` needs to use something else to ensure it's idempotent as it's a bit misleading for a function named like this to have side effects.